### DISCUSSION TITLE
Keeping tabs on which sensors are being received

### DISCUSSION THREAD

**Discussion by homeautomaton** [merbanan/rtl_433#2456] [Show and tell]
I have temperature sensors indoors, outdoors, in every fridge and freezer, plus a 5n1 weather station. Keeping up with putting in fresh (recharged) batteries, and making sure reception is okay (especially problematic from fridges/freezers) is a bit of a chore.

My rtl_433 data goes to NodeRED via MQTT, where it's filtered (only devices I want to follow pass through) and then sent to Home Assistant. The dashboard shown here is in HA, looking at the update times of sensors I expect to report frequently and letting me know which ones have fallen behind.

<img width="436" alt="image" src="https://user-images.githubusercontent.com/48721345/227780213-6a0edb97-0c3e-4c8f-a45c-3c7c50e71dd9.png">

**Comment by zuckschwerdt** [comment]
This is a nice setup. So no custom bridge script needed, all filtering and logic is in NodeRED? This seems very user friendly to set up.

**Reply by homeautomaton** [reply]
The rtl_433 mqtt messages processed in NodeRED are filtered based on settings stored in a sqlite database, and those I want to monitor are sent to a tcp/ip port that Home Assistant thinks is RFLink. I have a mixture of RFLink and translated rtl_433 protocols presently sent through this way. Over time, I'll probably transition everything I can to rtl_433, since RFLink seems to be languishing.
